agent/discovery/collector/client.go

```diff
@@ -35,6 +36,8 @@ type Config struct {
 	CA            string
 }
 
+var UuidNamespace = uuid.Must(uuid.Parse("fb92284e-aa5e-47f6-a883-bf9469e7a0dc"))
```

Comment:
I do not really like it here, I see this at a higher level in our modules, but I am getting bitten by cyclic imports.
Gonna keep it here for now if that's okay. :innocent:

Reply:
does moving this to `internal/uuid.go` helps? 
I would call it `TrentoNamespace` or something similar to  make it clear that is our defined namespace

Reply:
Deal.